Targets/AppStoreConnectUI/Sources/Authorization/AuthViewModel.swift

```diff
@@ -7,26 +7,34 @@
 
 import Foundation
 
-final class AuthViewModel: ObservableObject {
+public final class AuthViewModel: ObservableObject {
     private let authorization: Authorization
     
     @Published var issuerID = ""
-    @Published var privateKeyID = ""
-    @Published var privateKey = ""
+    @Published var privateKeyFileURL: URL? = nil
     
-    init(authorization: Authorization) {
+    public init(authorization: Authorization) {
         self.authorization = authorization
     }
     
-    func authorize() async throws {
-        try await authorization.authorize(issuerID: issuerID,
-                                          privateKeyID: privateKeyID,
-                                          privateKey: privateKey)
+    func authorize() async {
+        guard let privateKeyFileURL,
+              !issuerID.isEmpty
+        else {
+            assertionFailure("privateKeyFileURL is not set or issuerID is empty")
+            return
+        }
+
+        do {
+            try await authorization.authorize(issuerID: issuerID,
+                                              privateKeyFileURL: privateKeyFileURL)
+        }
+        catch {
+            print("\(error)")
```

Comment:
We can add some kind of logger later, let's add static func or static enum logger and use it everywhere instead of prints? 

```suggestion
static func log(_ error: Error) {
print(error)
}

static func log(message: String) {
print(error)
}
            log(error)
```